src/mixins/filing-template-mixin.ts

```diff
@@ -677,7 +645,7 @@ export default class FilingTemplateMixin extends DateMixin {
     ))
 
     // store Provisions Removed
-    this.setProvisionsRemoved(filing.alteration.provisionsRemoved)
+    if (filing.alteration.provisionsRemoved) this.setProvisionsRemoved(true)
```

Comment:
If the flag is not True then this does not update the initial value in the store. This is good because it doesn't break any code that checks if the store has changed (and in this case there is such a method, but untouched in this PR).